packages/ui/src/Slider.tsx

```diff
@@ -0,0 +1,343 @@
+          <input
+            ref={secondRangeInputRef}
+            type="range"
+            value={secondValue}
+            min={min}
+            max={max}
+            name={name}
+            disabled={disabled}
+            onChange={setSecondValueCallback}
+            step={step}
+            className={cn({
+              sliderClassName,
+              // The second thumb is above the first thumb in DOM.
+              // There is a corner case where both thumbs are above each other at the end
+              // of the range. In that case we need to pass pointer-events to the first thumb
+              // because that is the only one that can be moved.
+              [styles.atTheBorder]: max === secondValue && secondValue === firstValue,
+            })}
+            aria-labelledby={idRef.current}
+            aria-valuenow={secondValue}
+            aria-valuemin={Math.max(min, firstValue)}
+            aria-valuemax={max}
+            aria-invalid={!!error}
+            aria-errormessage={error && errorId(idRef.current)}
+          />
+        )}
+        <div className={styles.fillPlaceholder}>
+          <div
+            className={styles.fill}
+            data-test="fill"
+            // Since there is no easy way hot to pass properties to SCSS, we need to use
+            // inline styles.
+            style={
+              isRange
+                ? {
+                    width: `${width}%`,
+                    marginLeft: `calc(${left * 100}%)`,
+                  }
+                : {
+                    width: `calc(${max * left}%)`,
+                  }
+            }
+          />
+          {/*
+            Markers
+           */}
+          {!!marks?.length && (
+            <ul className={styles.marks} data-test="marks">
+              {marks.map(({ value: markValue }) => {
+                const { isActive, left } = getMarkMetadata(markValue, max, [firstValue, secondValue], isRange)
+                return (
+                  <li
+                    key={markValue}
+                    style={{ left: `${left}%` }}
+                    className={cn({
+                      [styles.active]: isActive,
+                    })}
+                  />
+                )
+              })}
+            </ul>
+          )}
+        </div>
+        {/*
+          Marks Descriptions
+        */}
+        {!!marks?.length && (
+          <ul className={styles.markDescriptions} data-test="mark-descriptions">
+            {marks.map(({ label, value: markValue }) => {
+              const { left } = getMarkMetadata(markValue, max, [firstValue, secondValue], isRange)
+              return (
+                <li key={markValue} style={{ left: `${left}%` }}>
+                  {label}
+                </li>
+              )
+            })}
+          </ul>
+        )}
+        {helperText && <Help parentId={idRef.current} helperText={helperText} className={styles.helperText} />}
```

Comment:
Last label seems kind of shifted
![image](https://user-images.githubusercontent.com/12794628/100324175-e7e5c000-2fd7-11eb-9d43-cc1af9c3b15c.png)

Reply:
Should we make the first and the last labels aligned with the slider left and right borders?

Reply:
I've discussed this with @pawelklasa, but I'm open to change it if he says it should look differently.

Reply:
It's all good, I think the only slight addition might be making the selection label bold